mpifileutils.spec

```
@@ -48,7 +48,7 @@
 
 Name:		mpifileutils
 Version:	0.11
-Release:	6%{?git_short:.g%{git_short}}%{?dist}
+Release:	8%{?git_short:.g%{git_short}}%{?dist}
```

Overall the `6` needs to become `7`
```suggestion
Release:	7%{?git_short:.g%{git_short}}%{?dist}
```